velox/core/QueryConfig.h

```diff
@@ -95,7 +94,7 @@ class QueryConfig {
       "abandon_partial_aggregation_min_pct";
 
   static constexpr const char* kMaxPartitionedOutputBufferSize =
-      "driver.max-page-partitioning-buffer-size";
+      "max_page_partitioning_buffer_size";
```

Comment:
Do you want to change it to be the same as the variable name, max_partitioned_output_buffer_size? max_partitioned_output_buffer_size is more understandable to me.